modules/hcp-disable-metrics.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-prepare/hcp-override-resource-util.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-disable-metrics_{context}"]
+= Disabling the metric service monitoring
+
+After enabling the `hypershift-addon` managed cluster addon, the metric service monitoring is configured by default so that {product-title} monitoring can gather metrics from `hypershift-addon`. 
+
+.Procedure
+
+You can disable the metric service monitoring by completing the following steps:
+
+. Log in to your hub cluster by running the following command:
++
+[source,terminal]
+----
+$ oc login
+----
+
+. Open the `hypershift-addon-deploy-config` add-on deployment configuration specification for editing by running the following command:
++
+[source,terminal]
+----
+$ oc edit addondeploymentconfig hypershift-addon-deploy-config -n multicluster-engine
+----
+
+. Add the `disableMetrics=true` customized variable to the specification, as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: addon.open-cluster-management.io/v1alpha1
+kind: AddOnDeploymentConfig
+metadata:
+  name: hypershift-addon-deploy-config
+  namespace: multicluster-engine
+spec:
+  customizedVariables:
+  - name: hcMaxNumber
+    value: "80"
+  - name: hcThresholdNumber
+    value: "60"
+  - name: disableMetrics <1>
+    value: "true"
+----
++
+<1> The `disableMetrics=true` customized variable disables the metric service monitoring configuration for both new and existing `hypershift-addon` managed cluster add-ons.
+
+. Save the changes.
```

Comment:
Maybe update this command instead to apply the changes to the YAML?
```suggestion
. Apply the changes to the configuration specification by running the following command:
+
[source,terminal]
----
$ oc apply -f <filename>.yaml
----
```

Follow up question that may be out of scope here, but do any services or clusters etc need to be restarted for the changes to take effect?

Reply:
Good question! I don't think so. Engineering and QE approved this text, and they didn't mention that anything else needed to be restarted for the changes to take effect.